tests/benchmark_text.nim

```diff
@@ -1,16 +0,0 @@
-import benchy, pixie
```

Comment:
why remove?

Reply:
benchmark_fonts already exists and tests this, i just forgot / duplicated at some point